libs/api/domains/auth-admin/src/lib/tenant/tenants.service.ts

```diff
@@ -147,4 +147,46 @@ export class TenantsService {
       pageInfo: { hasNextPage: false },
     }
   }
+
+  async getTenantById(id: string, user: User): Promise<Tenant> {
+    const tenants = await Promise.all([
+      this.adminDevApiWithAuth(user)
+        ?.meTenantsControllerFindById({ id })
+        .catch(this.handleError.bind(this)),
+      this.adminStagingApiWithAuth(user)
+        ?.meTenantsControllerFindById({ id })
+        .catch(this.handleError.bind(this)),
+      this.adminProdApiWithAuth(user)
+        ?.meTenantsControllerFindById({ id })
+        .catch(this.handleError.bind(this)),
+    ])
+
+    const tenantMap = new Map<string, TenantEnvironment[]>()
```

Comment:
This should just be an simple `TenantEnvironment[]` as we are just working with a single tenant and don't need a map of multiple tenant ids.